activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ServiceTaskConsumerHandler.java

```diff
@@ -199,7 +198,6 @@ private void assertIntegrationContextHeaders(IntegrationRequest integrationReque
                   .containsEntry(PROCESS_INSTANCE_ID, integrationContext.getProcessInstanceId())
                   .containsEntry(PROCESS_DEFINITION_ID, integrationContext.getProcessDefinitionId())
                   .containsEntry(APP_NAME, integrationRequest.getAppName())
-                  .containsEntry(APP_VERSION, integrationRequest.getAppVersion())
```

Comment:
This should not be removed, the integration request should have app version as a header.